include/seqan3/alignment/configuration/detail.hpp

```diff
@@ -34,6 +32,7 @@ enum struct align_config_id : uint8_t
     on_result,    //!< ID for the \ref seqan3::align_cfg::on_result "on_result" option.
     parallel,     //!< ID for the \ref seqan3::align_cfg::parallel "parallel" option.
     result,       //!< ID for the \ref seqan3::align_cfg::result "result" option.
+    result_type,  //!\brief ID for the \ref seqan3::align_cfg::detail::result_type "result_type" option.
```

Comment:
This is different from the other lines... I don't know what's the impact, but please consider to keep the existing style.
```suggestion
    result_type,  //!< ID for the \ref seqan3::align_cfg::detail::result_type "result_type" option.
```

Reply:
Attention: Here is `detail` code documentation in the user visible documentation!
@ second reviewer: Is it fine or should we avoid this? (how?)